### PR TITLE
feat: support precompiled-contract-address in relayer

### PR DESCRIPTION
**Pull request by mmsqe**

### CODE REVIEW

File: pystarport/cluster.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "autostart": "true",
     "autorestart": "true",
     "redirect_stderr": "true",
-    "startsecs": "3",
+    "startsecs": "1",
```
Comment (yihuang) [inline pystarport/cluster.py:36]: what's this for?

Comment (mmsqe) [reply pystarport/cluster.py:36]: for fasten test, actually I'm not sure why need wait 3s

Comment (yihuang) [reply pystarport/cluster.py:36]: Does it slowdown anything, I think the test logic just wait for port or block availability

Comment (mmsqe) [reply pystarport/cluster.py:36]: you is right

**Review by yihuang — APPROVED** [review]

**Pull request merged**